ts/nix.ts

```diff
@@ -163,6 +170,21 @@ export function nixStrLit(
   };
 }
 
+/**
+ * Returns a `NixExpression` which represents a dependency of the file file
+ * this expression is embedded in.
+ */
+export function nixFlakeDep(name: string, dep: FlakeDep): NixExpression {
```

Comment:
```suggestion
/**
 * Returns a `NixExpression` that renders as an identifier that refers to a flake input. At the same time it registers the flake input as a dependency, so that it'll be included in the inputs of the generated flake file. See `renderFlakeFile` for an example.
 */
export function nixFlakeDep(name: string, dep: FlakeDep): NixExpression {
```